frontend/src/home/Home.tsx

```diff
@@ -492,6 +511,11 @@ class HomeComponent extends React.Component<Props, HomeState> {
     this.props.setDNDSchedule(dndschedule);
   }
 
+  logOut() {
+    this.updatePlan();
+    this.props.logOut();
```

Comment:
I'm getting an error on logout because we're trying to call `this.props.updateActiveSchedule` after the logOut has been run. What I would recommend is to convert `this.updatePlan()` to an async function and use `await` instead of `.then`. That way in `logOut()` we can do 
```suggestion
    this.updatePlan().then(() => {
      this.props.logOut();
    });
```